Branch-SDK/src/main/java/io/branch/referral/TrackingController.java

```diff
@@ -60,8 +60,9 @@ private void onTrackingDisabled(Context context) {
     }
     
     private void onTrackingEnabled() {
-        if (Branch.getInstance() != null) {
-            Branch.getInstance().registerAppInitWithoutIntent();
+        Branch b = Branch.getInstance();
+        if (b != null) {
```

Comment:
We should avoid one letter variable names, even when it is obvious what is going on here.

Reply:
👍